### PR TITLE
Fixed issue #385

### PR DESCRIPTION
**Pull request by user-promi**

### CODE REVIEW

File: classes/class-mvx-payment-gateway.php [hunk 1]
```diff
@@ -187,7 +187,7 @@ public function email_notify($commission_status = 'mvx_processing') {
                     $email_vendor = WC()->mailer()->emails['WC_Email_Vendor_DirectBank_Commission_Transactions'];
                     $email_vendor->trigger($this->vendor->term_id);
                 }
-                if(!current_user_can('administrator')) :
+                if(!current_user_can('administrator') && $commission_status == 'mvx_processing') :
```
Comment (moumitahalder) [inline classes/class-mvx-payment-gateway.php]: checking missing

**Review by moumitahalder — CHANGES_REQUESTED** [review]

File: classes/class-mvx-payment-gateway.php (hunk 1 above)
Comment (abhirup97) [reply classes/class-mvx-payment-gateway.php]: Still indentation issue

**Comment by abhirup97** [comment]
@Sangi0  test this and add tested label if working fine

File: classes/class-mvx-payment-gateway.php [hunk 2]
```diff
@@ -187,8 +187,8 @@ public function email_notify($commission_status = 'mvx_processing') {
                     $email_vendor = WC()->mailer()->emails['WC_Email_Vendor_DirectBank_Commission_Transactions'];
                     $email_vendor->trigger($this->vendor->term_id);
                 }
-                if($commission_status) {
-                    if(!current_user_can('administrator') && $commission_status == 'mvx_processing') :
+                if ($commission_status) {
```
Comment (moumitahalder) [inline classes/class-mvx-payment-gateway.php:190]: why extra line for checking?

**Review by moumitahalder — CHANGES_REQUESTED** [review]

**Comment by moumitahalder** [comment]
do a fresh commit @user-promi

**Pull request closed without merging**